sega_genesis_checksum_utility_for_python_3.py

```diff
@@ -142,12 +142,12 @@ def verify_console_name_from_header(open_file):
     True if the header of `open_file` contains the correct console name in the
     right location.
   """
-  CONSOLE_NAME_OFFSET = 0x100
-  open_file.seek(CONSOLE_NAME_OFFSET)
-  console_name = open_file.read(15)
+  memorybuffer = open_file.read()
+  memorybuffer[0x100:0x10c].decode('utf-8')
+  console_name = memorybuffer[0x100:0x10c].decode('utf-8')
   if console_name == "SEGA MEGA DRIVE":
     return True
-  if console_name[:12] == "SEGA GENESIS":
```

Comment:
Conceptual question: did any of this logic need to change? Isn't it still functional in python3?

Reply:
I am removing my current logic and using another logic. `if console_name.strip() == ""SEGA GENESIS"" `  I am using strip method to remove the whitespace before and after the console name. It is REQUIRED because the console name might contain whitespace and don't execute the logic.

Reply:
Seems reasonable. Tested it out locally and seems to work fine.